docs/APIClient.md

```diff
@@ -203,6 +203,35 @@ aggregation.name = 'TopDomains'
 aggregation.save()
 ```
 
+### Sigma rules
+
+### Get a single rule
+
+To get a Sigma rule that is stored on the server via uuid of the rule:
+
+```
+rule = ts.get_sigma_rule("5266a592-b793-11ea-b3de-0242ac130004")
+```
+
+An example rule return is:
```

Comment:
the return here should be an object, that you can then do something with, eg.
```

rule.to_pandas()
```

or 

```
rule.to_json()
```

and then this would be the outptu